EstateReporting.BusinessLogic/IReportingManager.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace EstateReporting.BusinessLogic
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Models;
+    using Repository;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface IReportingManager
+    {
+        #region Methods
+
+        /// <summary>
+        /// Gets the transactions for estate.
+        /// </summary>
+        /// <param name="estateId">The estate identifier.</param>
+        /// <param name="startDate">The start date.</param>
+        /// <param name="endDate">The end date.</param>
+        /// <param name="groupingType">Type of the grouping.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        Task<TransactionsByDayModel> GetTransactionsForEstate(Guid estateId,
+                                                              String startDate,
+                                                              String endDate,
+                                                              GroupingType groupingType,
+                                                              CancellationToken cancellationToken);
+
+        #endregion
+
+        //Task<TransactionsByDayModel> GetTransactionsForMerchant(Guid estateId,
+        //                                                        Guid merchantId,
+        //                                                        DateTime startDate,
+        //                                                        DateTime endDate,
+        //                                                        GroupingType groupingType,
+        //                                                        CancellationToken cancellationToken);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6325301)